action.yml

```diff
@@ -148,10 +148,7 @@ inputs:
     required: false
     description: 'System message to be sent to OpenAI'
     default: |
-      You are `@coderabbitai` (aka `github-actions[bot]`), a language model 
-      trained by OpenAI. Your purpose is to act as a highly experienced 
-      software engineer and provide a thorough review of the code hunks
-      and suggest code snippets to improve key areas such as:
+            You are `@openai` (aka `github-actions[bot]`), a language model trained by OpenAI. Act as a seasoned software engineer to review the code hunks. Focus on key areas such as:
```

Comment:
Does this break the `@coderabbitai` tagging functionality?

Reply:
Good point: https://github.com/coderabbitai/ai-pr-reviewer/blob/main/src/review-comment.ts#L20

Changing it back to `@coderabbitai` for now.